extension/src/tdigest.rs

```diff
@@ -245,6 +251,48 @@ CREATE AGGREGATE timescale_analytics_experimental.tdigest(size int, value DOUBLE
 );
 "#);
 
+#[pg_extern(schema = "timescale_analytics_experimental")]
+pub fn tdigest_compound_trans(
+    state: Option<timescale_analytics_experimental::TDigest<'static>>,
```

Comment:
I'd probably have the internal state still be an InternalTDigest, and let the value be the PG TDigest type.

Reply:
It seems like converting between the different forms could be bad, and also, it would make it easier to modify in place?

Reply:
Agreed, this behaves much more reasonably as a InternalTDigest.